x-pack/plugins/cases/server/telemetry/queries/alerts.test.ts

```diff
@@ -42,13 +43,23 @@ describe('alerts', () => {
         savedObjectsClient: telemetrySavedObjectsClient,
         logger,
       });
+
       expect(res).toEqual({
         all: {
           total: 5,
           daily: 3,
           weekly: 2,
           monthly: 1,
           maxOnACase: 1,
+          counts: {
+            buckets: [
+              { doc_count: 1, key: 1 },
+              { doc_count: 2, key: 2 },
+              { doc_count: 3, key: 3 },
+            ],
+          },
+          references: { cases: { max: { value: 1 } } },
+          additionalAggsResult: { value: 5 },
```

Comment:
I hadn't realized before this was happening. We don'r need to return these additional fields here. The return type of `getAlertsTelemetryData` is `comments: { all: Count & { maxOnACase: number } }` where count is:
```
{
  total: number;
  monthly: number;
  weekly: number;
  daily: number;
}
```

So the fields `counts`, `references`, and  `additionalAggsResult` are unnecessary.

Only the value returned in `total` needs to be updated.

Maybe we could do some cleanup in `getAlertsTelemetryData` 🤔 wdyt?

Reply:
done, thanks!